plugin/ios/AnylineReact.podspec

```diff
@@ -18,7 +18,7 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/Anyline/anyline-ocr-react-native-module.git", :tag => "#{s.version}" }
 
   s.source_files  = "*.{h,m}"
-  s.dependency "Anyline", "~> 52"
+  s.dependency "Anyline", "~> 53"
```

Comment:
@vimalvenugopalananyline Please replace

"~> 53"

with

"53.0.0"

to lock to specific versions instead of major versions

Reply:
Fixed